articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Tutorial: Migrate MySQL offline to Azure Database for MySQL"
+titleSuffix: "Azure Database Migration Service"
+description: "Learn to perform an offline migration from MySQL on-premises to Azure Database for MySQL by using Azure Database Migration Service."
+services: dms
+author: "sumitgaurin"
+ms.author: "sgaur"
+manager: "arthiaga"
+ms.reviewer: "arthiaga"
+ms.service: dms
+ms.workload: data-services
+ms.custom: "seo-lt-2019"
+ms.topic: tutorial
+ms.date: 04/11/2021
+---
+
+# Tutorial: Migrate MySQL to Azure Database for MySQL offline using DMS
+
+You can use Azure Database Migration Service to perform a one-time full database migration on-premises MySQL instance to [Azure Database for MySQL](../mysql/index.yml) with high speed data migration capability. In this tutorial, we will migrate a sample database from an on-premises instance of MySQL 5.7 to Azure Database for MySQL (v5.7) by using an offline migration activity in Azure Database Migration Service. Although the articles assumes the source to be a MySQL database instance and target to be Azure Database for MySQL, it can be used to migrate from one Azure Database for MySQL to another just by changing the source server name and credentials. Also, migration from lower version MySQL servers (v5.6 and above) to higher versions is also supported.
```

Comment:
Our offline scenario should let you do version downgrades as well. Although im not sure we want to promote this.

Reply:
That is the reason I have explicitly mentioned the move forward. We should not promote the downgrade path. Just my thought, but open to suggestions.

Reply:
I would vote to leave out the mention for now.